spec.html

```diff
@@ -23527,7 +23527,8 @@
     <!-- es6num="19.3.3" -->
     <emu-clause id="sec-properties-of-the-boolean-prototype-object">
       <h1>Properties of the Boolean Prototype Object</h1>
-      <p>The Boolean prototype object is the intrinsic object %BooleanPrototype%. The Boolean prototype object is an ordinary object. It is not a Boolean instance and does not have a [[BooleanData]] internal slot.</p>
```

Comment:
I think confusion is: 

> The Boolean prototype object is the intrinsic object %BooleanPrototype%. The Boolean prototype object is an ordinary object

Seems like it's saying contradictory things, but isn't.

I'd recommend a complete change to: 

"The Boolean prototype object is itself a Boolean object, it has a [[BooleanData]] internal slot with the value false."